geo-replication/syncdaemon/primary.py

```diff
@@ -1656,7 +1656,7 @@ def register(self, register_time=None, status=None):
         self.counter = 0
         self.comlist = []
         self.stimes = []
-        self.sleep_interval = 60
+        self.sleep_interval = gconf.get("change-interval")
```

Comment:
It is better to have a default value for sleep_interval. If the change-interval is not specified in the conf file use a default.

Reply:
Thanks for the review. I added default values to the different calls to change-interval config.